README.md

```diff
@@ -3,11 +3,17 @@
   <a href="https://mui.com/" rel="noopener" target="_blank"><img width="150" src="https://mui.com/static/logo.svg" alt="MUI logo"></a>
 </p>
 
-<h1 align="center">MUI</h1>
+<h1 align="center">MUI Core</h1>
 
-<div align="center">
+**MUI Core** contains foundational React UI component libraries for shipping new features faster.
+
+- _Material UI_ is a comprehensive library of components that features our implementation of Google's [Material Design](https://material.io/design/introduction/) system.
+
+- _MUI Base_ is our library of "unstyled" components and low-level hooks. With Base, you have complete control over your app's CSS.
```

Comment:
```suggestion
- _MUI Base_ is our library of "unstyled" components and low-level hooks. With Base, you have complete control over your app's CSS while gaining accessibility features support out-of-the-box.
```

My suggestion might be a bit wordy so feel free to improve! I do feel like accessibility is a major offered value in terms of Base and we should probably mention it as much as we can :)

Reply:
How about something simple like this?

```suggestion
- _MUI Base_ is our library of "unstyled" components and low-level hooks. With Base, you gain complete control over your app's CSS and accessibility features.
```

Reply:
Yup! That's great!